matRad_calcDoseInfluence.m

```diff
@@ -0,0 +1,50 @@
+function dij = matRad_calcDoseInfluence(ct,cst,stf,pln)
+% matRad dose calculation automaticly creating the appropriate dose engine
+% for the given pln struct and called the associated dose calculation funtion
+%
+% call
+%   dij =  matRad_calcDoseInfluence(ct,stf,pln,cst)
```

Comment:
Minor comment: order of the inputs to be adjusted, when using 'help matRad_calcDoseInfluence'